Callalen/HikingChallenge.java

```diff
@@ -79,7 +85,7 @@ public static void main(String[] args) throws IOException {
                 time += path.get(i).h;
                 out.println(path.get(i));
             }
-            out.printf("Your time to beat is %d minutes.\n", time);
+            out.printf("Your time to beat is %d minutes.\n", current.g-current.h);
```

Comment:
the current node's g value is the cost to get from start THROUGH that node...so subtract the cost to get through the current node from the total cost to get the cost to get TO that node